thirdparty/pip/requirements-per-platform.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/defs.bzl b/defs.bzl
+index 9a327f6..721f7a7 100644
+--- a/defs.bzl
++++ b/defs.bzl
+@@ -4,6 +4,24 @@ load("@rules_python//python:defs.bzl", "py_test")
+ 
+ pip_vendor_label = Label("@com_github_ali5h_rules_pip//:third_party/py/easy_install.py")
+ 
++def _get_platform(repository_ctx):
++    if "mac" in repository_ctx.os.name:
++        return "osx"
++
++    return repository_ctx.os.name
++
++def _select_requirements_for_platform(repository_ctx):
++    current_platform = _get_platform(repository_ctx)
++
++    for label, intended_platform in repository_ctx.attr.requirements_per_platform.items():
++        if intended_platform == current_platform:
++            return repository_ctx.path(label)
```

Comment:
totally agree, this code is from https://github.com/128technology/rules_pip/blob/master/rules/repository.bzl#L7

which I think was done that way to utilize `label_keyed_string_dict` and enforce the labels. It doesn't seem like there's an inverse... `label_value_string_dict`

Reply:
Hm, I wonder if there's a way to just call `Label` constructor in the impl and it will automatically fail if it's not a valid label... but yeah whatever, not a big deal either way